setup.py

```diff
@@ -35,6 +35,8 @@
     "pandas",  # previously pinned to 2.0.3
     "polars==0.17.2",  # 04/18/23 this breaks our CI
     "pyarrow",
+    "pyspark",
```

Comment:
seems like these two are only required for the integration tests, please move to this file: https://github.com/ploomber/jupysql/blob/4fda1656df105b296cf684b2fab92f47802e96ee/noxfile.py#L32

Reply:
have one unit test that requires it, I'll see if I can refactor it

Reply:
Should be resolved refactored is_spark function

Reply:
cool, just add a note explaining that these two dependencies are required for spark integration testing

Reply:
https://github.com/ploomber/jupysql/pull/968/commits/db024fb2149b3b050f6ea0507c4cc36a3da77cfc#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R74